pootle/apps/pootle_app/management/commands/update_tmserver.py

```diff
@@ -200,8 +201,8 @@ def _initialize(self, *args, **options):
         try:
             self.tm_settings = settings.POOTLE_TM_SERVER[options.get('tm')]
         except KeyError:
-            raise CommandError('Specified Translation Memory is not defined '
-                               'in POOTLE_TM_SERVER.')
+            raise CommandError("Translation Memory '%s' is not defined in "
+                               "POOTLE_TM_SERVER setting." % options.get('tm'))
```

Comment:
So what about inviting users to review their settings as well? Something like *"Translation Memory '%s' is not defined in the POOTLE_TM_SERVER setting. Please ensure it exists and double-check you typed it correctly."*